projects/plugins/jetpack/extensions/blocks/subscriptions/deprecated/v6/index.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Internal dependencies
+ */
+import definedAttributes from '../v3/attributes';
+import save from './save';
```

Comment:
Conceptually, the attributes and save method should be "identical" to what was present before the changes/deprecation occured. So I think here it'd be most correct to add a new `v6/attributes.js` file (that mirrors the attributes before adding the successMessage), and in the `v6/save.js` to import those attributes and mirror the `save.js` implementation before you started making changes - imports may need adjusting (e.g. the constants).

Reply:
Hi, I have created a new file `attributes.js`. I don't need to change any constants. :)